src/com/pty4j/unix/UnixPtyProcess.java

```diff
@@ -58,7 +57,7 @@ public UnixPtyProcess(@NotNull PtyProcessOptions options, boolean consoleMode) t
     myConsoleMode = consoleMode;
     myPty = new Pty(consoleMode, options.isUnixOpenTtyToPreserveOutputAfterTermination());
     myErrPty = options.isRedirectErrorStream() ? null : (consoleMode ? new Pty() : null);
-    String dir = MoreObjects.firstNonNull(options.getDirectory(), ".");
+    String dir = PtyUtil.firstNonNull(options.getDirectory(), ".");
```

Comment:
Please use `Objects.requireNonNullElse`

Reply:
Here and in other similar places in the PR.

Reply:
@segrey Alright I was not aware of this method